util/scripts/install.sh

```diff
@@ -0,0 +1,9 @@
+echo
+echo installing...
```

Comment:
Can you echo a path where the libraries are installed?

Reply:
that is echoed a the end of the install:

```
Install the project...
-- Install configuration: "Release"
-- Installing: /home/dpetre/.local/lib/python3.5/site-packages/py3d.cpython-35m-x86_64-linux-gnu.so
```